.drone.yml

```diff
@@ -69,6 +68,7 @@ steps:
       GO111MODULE: on
       GOPROXY: off
     commands:
+      - curl -sL https://deb.nodesource.com/setup_12.x | bash - && apt install -y nodejs
```

Comment:
I'm not familiar with these things but maybe we need to allow for some proxy configuration here? I've heard that we sometimes have problems grabbing packages when we build from servers located in China.

Reply:
those fetches should only happen on the CI. I'm not sure if great firewall applies there as well.

`curl` supports `HTTPS_PROXY` so users can set that to their liking beforhand.

Reply:
It looks like this can be addressed in a future PR if required as we currently don't have a need for the proxy. 😉

Reply:
The fact the the build currently passes tells me no proxy is required. I guess we want to set a CI-wide `HTTPS_PROXY` variable at some point if required because alpine-based containers also need to fetch various dependencies.